examples/auth/README.md

```diff
@@ -10,8 +10,8 @@ This directory contains a GraphQL server (based on [`graphql-yoga`](https://gith
 .
 ├── README.md
 ├── database
-│   └── datamodel.graphql
-├── graphcool.yml
+│   ├── schema.graphql
```

Comment:
Please rename this file to `datamodel.graphql` (also adjust in `.graphqlconfig.yml`.